triggerReleasePipeline.sh

```diff
@@ -43,27 +43,28 @@ checkArgs $#
 # cleanup properties if exists from previous run
 git clean -fd # same as rm -rf ${SCRIPT_DIR}/properties but wont touch ${WORKSPACE}/tracking file
```

Comment:
this need to update.
if we do not have local repo, "git clean -fd " will fail

Reply:
This is for the mirror-scripts repo that contains the "properties" file, not the openjdk repo